File: TransCoder_tokenized_test_set_functions/java/CHECK_ARRAY_MAJORITY_ELEMENT.java
static boolean isMajority ( int a [ ], int n ) {
  HashMap < Integer, Integer > mp = new HashMap < Integer, Integer > ( );
  for ( int i = 0;
  i < n;
  i ++ ) if ( mp . containsKey ( a [ i ] ) ) mp . put ( a [ i ], mp . get ( a [ i ] ) + 1 );
  else mp . put ( a [ i ], 1 );
  for ( Map . Entry < Integer, Integer > x : mp . entrySet ( ) ) if ( x . getValue ( ) >= n / 2 ) return true;
  return false;
}

File: TransCoder_tokenized_test_set_functions/cpp/CHECK_ARRAY_MAJORITY_ELEMENT.cpp
bool isMajority ( int a [ ], int n ) {
  unordered_map < int, int > mp;
  for ( int i = 0;
  i < n;
  i ++ ) mp [ a [ i ] ] ++;
  for ( auto x : mp ) if ( x . second >= n / 2 ) return true;
  return false;
}